sigs.yaml

```diff
@@ -2474,3 +2428,51 @@ workinggroups:
     contact:
       slack: wg-k8s-infra
       mailing_list: https://groups.google.com/forum/#!forum/kubernetes-wg-k8s-infra
+usergroups:
+  - name: Big Data
+    dir: ug-big-data
+    mission_statement: >
+      Serve as a community resource for advising big data and data science related software projects
+      on techniques and best practices for integrating with Kubernetes.
+      Represents the concerns of users from big data communities to Kubernetes for the purposes of
+      driving new features and other enhancements, based on big data use cases.
+    charter_link:
+    label: big-data
+    leadership:
+      chairs:
+      - name: Anirudh Ramanathan
+        github: foxish
+        company: Rockset
+      - name: Erik Erlandson
+        github: erikerlandson
+        company: Red Hat
+      - name: Yinan Li
+        github: liyinan926
+        company: Google
+    meetings:
+    - description: Regular User Group Meeting
+      day: Wednesday
+      time: "18:00"
+      tz: "UTC"
+      frequency: biweekly
+      url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+      archive_url: https://docs.google.com/document/d/1pnF38NF6N5eM8DlK088XUW85Vms4V2uTsGZvSp8MNIA/edit
+      recordings_url: https://docs.google.com/document/d/1pnF38NF6N5eM8DlK088XUW85Vms4V2uTsGZvSp8MNIA/edit
+    contact:
+      slack: sig-big-data
```

Comment:
The slack channel + mailing list + teams need to be renamed.

For renaming teams, can you create a PR here: https://github.com/kubernetes/org/tree/master/config/kubernetes/sig-big-data

Reply:
slack channel should be no prob. is there anything special about creating new mailing lists?

Reply:
> is there anything special about creating new mailing lists?

I think you can just rename the existing mailing list. This is breaking change though, since emails to the old mailing list address will bounce.

@dims and @spiffxp recently did it for `k8s-infra-team` -> `wg-k8s-infra` (https://github.com/kubernetes/k8s.io/issues/166) so they probably know all the details better. :)

Reply:
Unless people feel otherwise, my plan is to migrate the mailing lists and slack when these changes merge.